### ISSUE TITLE
Solar charger?

### ISSUE THREAD

**Issue by qnixAVA** [Xinyuan-LilyGO/T-A7670X#59]
Hi, which PIN is to get information about the solar charger?

**Comment by qnixAVA** [comment]
nothing new ?

**Comment by lewisxhe** [comment]
From the schematic diagram, it can be seen that the solar control is CN3065, with a maximum charging current limit of 500mA and a maximum input voltage of 4.4-6V

**Comment by qnixAVA** [comment]
Yep but is there any GPIO to get this information? voltage of my current charging source for example?

**Comment by lewisxhe** [comment]
No, this is completely hardware controlled and cannot be determined
